Dryad Digital Repository.js

```diff
@@ -0,0 +1,227 @@
+
+{
+	"translatorID": "7a81d945-7d9c-4f8c-bd7b-4226c1cab40e",
+	"label": "Dryad Digital Repository",
+	"creator": "Nathan Day",
+	"target": "^https?://(?:www\\.)?datadryad\\.org/",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2014-11-09 20:04:09"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2014 Dryad Digital Repository
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	var xpath = '//meta[@name="DC.type"]';
+	if (ZU.xpath(doc, xpath).length > 0) {
+		if (ZU.xpath(doc, xpath) === "Article") {
+			return "Article";
+		} else if (ZU.xpath(doc, xpath) === "Dataset") {
+			return "Dataset";
+		}
+	}
+	return false;
+}
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) === "Article" || detectWeb(doc, url) === "Dataset") {
+		var translator = Zotero.loadTranslator("import");
+		// use the Embedded Metadata translator
+		translator.setTranslator("951c027d-74ac-47d4-a107-9c3069ab7b48");
+		translator.setHandler("itemDone", function (obj, item) {
+			item.abstractNote = item.extra;
+			item.extra = '';
+			item.complete();
+		});
+		translator.getTranslatorObject(function (obj) {
+			obj.doWeb(doc, url);
+		});
```

Comment:
I'd like you to address this. While it works, `.getTranslatorObject` is rather complex due to sandboxing and we should avoid it as much as possible.